src/client/feed/HeroBannerContainer.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+import HeroBanner from '../components/HeroBanner';
+import { getIsAuthenticated, getIsLoaded, getIsBannerClosed } from '../reducers';
+import { closeBanner } from '../app/appActions';
+
+export default connect(
```

Comment:
I really like this idea, having a suffix for a component that is `Container`, which handles all the state properties 👍